utils/wasm-builder/src/wasm_project.rs

```diff
@@ -221,6 +221,28 @@ pub const WASM_BINARY_META: &[u8] = include_bytes!("{}");
         let exports = vec!["init", "handle", "handle_reply", "__gear_stack_end"];
         pwasm_utils::optimize(&mut module, exports)
             .map_err(|_| BuilderError::UnableToOptimize(from.to_path_buf()))?;
+
+        let mut required_export_exist = false;
+
+        for entry in module
+            .export_section()
+            .ok_or_else(|| anyhow::format_err!("WASM module does't contain export section"))?
```

Comment:
I'd add all errors to the one struct: https://github.com/gear-tech/gear/blob/master/utils/wasm-builder/src/builder_error.rs